runtime/src/comm/gasnet/comm-gasnet.c

```diff
@@ -931,6 +931,14 @@ void chpl_comm_init(int *argc_p, char ***argv_p) {
 
   gasnet_set_waitmode(GASNET_WAIT_BLOCK);
 
+  gex_Rank_t count = -1;
```

Comment:
This is using the GASNet-EX API, but we're still using GASNet-1. We're looking to migrate to EX in https://github.com/chapel-lang/chapel/issues/14443, but I don't think we should bring in the EX API just for this. (We have some users who drop in an old GASNet-1 tarball for ofi support, which isn't in EX yet.) 

ETA for upgrading to EX is sometime this year, but not before the release.

Reply:
I didn't realize GASNet-EX isn't standard. I couldn't find a comparable GASNet-1 API. Maybe we don't support this functionality on GASNet-1?

Reply:
I wouldn't say it's not standard, we just haven't migrated to it yet. I would guess we just don't have this functionality in GASNet-1

Reply:
Would it work to make it  `__attribute__((weak))` as we do for e.g. `PMI_Barrier`? That way we use it if it's available, otherwise we'll just have to rely on the environment variable on GASNet-1.

Reply:
That could work, but I feel a little sheepish about that (mostly about portability of `__attribute__(weak)`. I'd be more comfortable if we held off on this until we upgraded to the EX API in general (which would unfortunately mean sitting on this PR for a few months.)

Curious for @gbtitus's take on this though

Reply:
I'd expect Gnu-style attribute syntax to work with all target compilers, and specifically `__attribute__((weak))` to be supported on all platforms, but I may be forgetting a case. What target compiler(s) do we compile comm=gasnet with, that we don't compiler comm=ofi with?